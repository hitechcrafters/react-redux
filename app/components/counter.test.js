import React from 'react';
import Counter from './Counter';
import renderer from 'react-test-renderer';

it('Renders correctly', () => {
  const component = renderer
    .create(<Counter />);

    expect(component).toMatchSnapshot();
});
