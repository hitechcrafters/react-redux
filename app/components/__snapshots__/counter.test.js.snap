// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="center"
>
  <div>
    <button
      className="btn"
      onClick={undefined}
    >
      +
    </button>
  </div>
  <h1
    className="margin-lft margin-rgt"
  />
  <div>
    <button
      className="btn"
      onClick={undefined}
    >
      -
    </button>
  </div>
</div>
`;
